enrich-points-with-boundaries/geodatabase/build.gradle

```diff
@@ -17,19 +17,24 @@ dependencies {
 
 	testImplementation "org.apache.spark:spark-core_$scala_version:$spark_version"
 	testImplementation "org.apache.spark:spark-sql_$scala_version:$spark_version"
-	testImplementation "org.scalatest:scalatest_$scala_version:2.2.6"
-	testImplementation 'junit:junit:4.12'
+	testImplementation "org.scalatest:scalatest_$scala_version:3.2.2"
+	testImplementation "org.scalatest:scalatest-funsuite_$scala_version:3.2.2"
 }
 
+task scalaTest(dependsOn: ['testClasses'], type: JavaExec) {
```

Comment:
Why did we change to using JavaExec here instead of keeping it as the normal test task?

Reply:
The test task used to work because scalatest had a JunitRunner to use. That JunitRunner was removed so the test task no longer found the scala tests

Reply:
The docs say that they just moved JUnitRunner to a new artifact and package, try adding this dependency with the old setup (oh and update junit to 4.13):

	testImplementation "org.scalatestplus:junit-4-13_$scala_version:3.2.2.0"